### ISSUE TITLE
The center progressbar looks stretched

### ISSUE THREAD

**Issue by timkoers** [panwrona/DownloadProgressBar#7]
I've added the View to my layout, and it works great. But the only problem is that the progressbar in the middle is not as it's supposed to be according to the gif animations.

EDIT: On Android 4.x.x

![stretched downloadprogressbar](https://cloud.githubusercontent.com/assets/4262725/11043086/471e3842-871b-11e5-89f1-55e9fc2b1921.png)

**Comment by panwrona** [comment]
In the center you have the oval shape. I think this is because of the dimensions you've put to the view. Can you paste here the xml code?

**Comment by timkoers** [comment]
The same dimensions as your sample app uses.

Van: Mariusz Brona notifications@github.com
verzonden: 10 nov. 2015 10:07
Aan: panwrona/DownloadProgressBar
Cc: timkoers
Onderwerp: Re: [DownloadProgressBar] The center progressbar looks stretched (#7)

In the center you have the oval shape. I think this is because of the dimensions you've put to the view. Can you paste here the xml code?

---

**Comment by panwrona** [comment]
Are You sure? Cause it doesn't seem to be 1dp stroke width

**Comment by timkoers** [comment]
Yes I'm very sure. Here is my code: http://pastebin.com/5Q6aPvaz

**Comment by timkoers** [comment]
The image itself is zoomed in.

**Comment by panwrona** [comment]
Hmm, ok I'll try to make situation like this, but I need You to tell me what device You use and android version installed on this device

**Comment by timkoers** [comment]
I use the Android emulgator with Android version 4.x.xx, it also happens on the HTC One X running Android 4.2.2

Van: Mariusz Brona notifications@github.com
verzonden: 12 nov. 2015 11:22
Aan: panwrona/DownloadProgressBar
Cc: timkoers
Onderwerp: Re: [DownloadProgressBar] The center progressbar looks stretched (#7)

Hmm, ok I'll try to make situation like this, but I need You to tell me what device You use and android version installed on this device

---
